demo/app/ios/Runner/flutterPlugin.swift

```diff
@@ -367,6 +366,15 @@ public class SwiftWalletSDKPlugin: NSObject, FlutterPlugin {
                      claims["value"] = claim.value()
                      claims["rawValue"] = claim.rawValue()
                 }
+                var order: Int = -1
+                if claim.hasOrder() == true {
+                    do {
+                        try claim.order(&order)
```

Comment:
Question: if you do

```
claim.order(&order, nil)
```
does the compiler allow you to get rid of the try/catch? The reason I ask is because if `hasOrder` returns true then it's safe to skip checking the error for `order`.

Reply:
No it gives compile time error

Reply:
(see my documentation for more info: https://github.com/trustbloc/wallet-sdk/blob/main/cmd/wallet-sdk-gomobile/docs/usage.md#claim)